route/here/measure.go

```diff
@@ -94,13 +94,23 @@ func NewClient(apiKey string, opts ...ClientOption) Client {
 	return c
 }
 
+func cleanPoints(points []route.Point) []route.Point {
+	for i, p := range points {
+		if len(p) == 2 && p[0] == 0 && p[1] == 0 {
+			points[i] = route.Point{}
```

Comment:
Why don't we remove the points then vs. setting it to the zero-value struct?

Reply:
because then we change the matrix size from MxM to a smaller matrix and this matrix then cannot be used in router because it requires the MxM